editor/store/reducer.js

```diff
@@ -584,6 +584,18 @@ export function isTyping( state = false, action ) {
 	return state;
 }
 
+export function isAutosaving( state = false, action ) {
```

Comment:
Can we add JSDoc describing what state is being tracked here.

Reply:
added in https://github.com/WordPress/gutenberg/pull/6257/commits/4204ad2dba24e3acc0797b9ae11fb178d2a442cf